fp-proj/src/app/core/services/authorization.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { OAuthService, UserInfo, OAuthEvent, EventType } from 'angular-oauth2-oidc';
+import { HttpClient } from '@angular/common/http';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import {User} from '../../models/User'
+
+@Injectable({
```

Comment:
jesli rejestrujesz serwis w providers: [] w core module  to "providedIn: root" w dekoratorze nie jest potrzebne,